unison-src/transcripts/branch-command.output.md

```diff
@@ -18,40 +18,137 @@ someterm = 18
 ```
 Now, the `branch` demo:
 
-`branch` can create a branch from a different branch in the same project.
+`branch` can create a branch from a different branch in the same project, from a different branch in a different
+project, or from loose code.
 
 ```ucm
-foo/main> branch topic
+foo/main> branch topic1
 
-  Done. I've created the topic branch based off of main
+  Done. I've created the topic1 branch based off of main
   
-  Tip: Use `merge /topic /main` to merge your work back into the
-       main branch.
+  Tip: Use `merge /topic1 /main` to merge your work back into
+       the main branch.
 
-```
-`branch` can create a branch from a different branch in a different project.
+foo/main> branch /topic2
+
+  Done. I've created the topic2 branch based off of main
+  
+  Tip: Use `merge /topic2 /main` to merge your work back into
+       the main branch.
+
+foo/main> branch foo/topic3
+
+  Done. I've created the topic3 branch based off of main
+  
+  Tip: Use `merge /topic3 /main` to merge your work back into
+       the main branch.
+
+foo/main> branch main topic4
+
+  Done. I've created the topic4 branch based off of main
+  
+  Tip: Use `merge /topic4 /main` to merge your work back into
+       the main branch.
+
+foo/main> branch main /topic5
```

Comment:
Does `branch` switch you to the branch it just created? It should, but I can't tell if it does from this transcript output.

Reply:
It does, yeah. I couldn't think of a way to capture this in a transcript.